dev/kube/klog.sh

```diff
@@ -29,9 +29,17 @@ if [ "${FORCE}" == "1" ] ; then
   rm -f "${DONE}" 2> /dev/null
 fi
 
+function red() {
+  printf '\e[31m%b\e[0m\n' "$1"
+}
+
+function green() {
+  printf '\e[32m%b\e[0m' "$1"
+}
```

Comment:
Why does `red` include a new line, but `green` does not?

Is `red` even used?

Reply:
Oops. Will fix. And yes, look for `Already done`.

Reply:
Oh. I somehow lost that piece :(

Reply:
If you mean 4 lines lower… that's uncolourized.

Reply:
Fixed with commit 957cb317c81737bce63b4e6313bba145ab10d694